File: 프로그래머스/3/42884. 단속카메라/단속카메라.py
def solution(routes):
  routes.sort(key = lambda x: x[1])
  new_camera_position = -30001
  answer = 0
  for route in routes:
    if new_camera_position < route[0]:
      answer += 1
      new_camera_position = route[1]
  return answer

File: 프로그래머스/3/42884. 단속카메라/단속카메라.js
function solution(routes) {
    routes.sort((a, b) => a[1] - b[1]);
    let min = -30001;
    let answer = 0;
    for (route of routes) {
        if (route[0] <= min) {
            continue;
        } else {
            answer += 1;
            min = route[1];
        }
    }
    return answer;
}